nodebuilder/p2p/module.go

```diff
@@ -91,8 +91,11 @@ func ConstructModule(tp node.Type, cfg *Config) fx.Option {
 					}
 				}
 
+				return allowlist, nil
+			}),
+			fx.Provide(func(ctx context.Context, allowlist []ma.Multiaddr) (network.ResourceManager, error) {
 				return rcmgr.NewResourceManager(
-					rcmgr.NewFixedLimiter(limits.AutoScale()),
+					rcmgr.NewFixedLimiter(rcmgr.DefaultLimits.AutoScale()),
```

Comment:
There is a bug related to limits here + plus unused code introduced. Check line 62

Reply:
Can you explain what bug was introduced? can you also please @Wondertan check on the new changes and lmk if that resolves it? (_made an assumption about what the bug is_)